axes/tests/test_management_commands.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import unicode_literals
+
+from io import StringIO
+from sys import version_info
+from unittest import skipIf
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from axes.models import AccessAttempt
+
+
+@skipIf(version_info < (3, ), 'Not supported on Python 2.x')
```

Comment:
Python 2.7 and Python 3.4 support have been removed now. Python 3.4 is deprecating very soon as well, and if we don't support 2.x anymore, it doesn't make much sense supporting 3.4 in the test matrices either, since Django has dropped Python 3.4 from the newer versions as well.